### PR TITLE
Add functional tests for /whatsnew pages (Fixes #8000)

### LINKED ISSUE #8000: Add functional tests for whatsnew pages
## Description

We have some tests for the legacy fallback page, but none for the current version-specific pages. We should have test coverage to make sure active whatsnew pages render correctly.

---

#### :yellow_heart: Success Criteria :yellow_heart:
- [ ] All active whatsnew pages have automated test coverage.
- [ ] Each test verifies that the requested page renders as expected (could include a check for a CTA button, for example).

#### :heavy_exclamation_mark: Risks :heavy_exclamation_mark:
- [ ] This is an example of a risk. Use a checklist!
- [ ] This is an example of a risk.

#### Tasks
- [ ] This is an example of a task. Use a checklist!
- [ ] This is an example of a task.

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
Adds functional tests for all current /whatsnew templates

## Issue / Bugzilla link
#8000

## Testing
Successful test run: https://ci.vpn1.moz.works/blue/organizations/jenkins/bedrock_multibranch_pipeline/detail/run-integration-tests/425/pipeline/

### CODE REVIEW

File: tests/pages/firefox/whatsnew/whatsnew_67_0_5.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+from pages.firefox.base import FirefoxBasePage
+
+
+class FirefoxWhatsNew6705Page(FirefoxBasePage):
+
+    URL_TEMPLATE = '/{locale}/firefox/67.0.5/whatsnew/all/{params}'
+
+    _firefox_accounts_form_locator = (By.ID, 'fxa-email-form')
+    _monitor_button_locator = (By.CLASS_NAME, 'js-monitor-button')
```
Comment (craigcook) [inline tests/pages/firefox/whatsnew/whatsnew_67_0_5.py]: This should change to `js-fxa-product-button` (that happened after this PR was made).

File: tests/pages/firefox/whatsnew/whatsnew_68.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+from pages.firefox.base import FirefoxBasePage
+
+
+class FirefoxWhatsNew68Page(FirefoxBasePage):
+
+    URL_TEMPLATE = '/{locale}/firefox/68.0/whatsnew/all/{params}'
+
+    _firefox_accounts_form_locator = (By.ID, 'fxa-email-form')
+    _monitor_button_locator = (By.CLASS_NAME, 'js-monitor-button')
```
Comment (craigcook) [inline tests/pages/firefox/whatsnew/whatsnew_68.py]: `js-fxa-product-button`

File: tests/pages/firefox/whatsnew/whatsnew_69.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+from selenium.webdriver.common.by import By
+
+from pages.firefox.base import FirefoxBasePage
+
+
+class FirefoxWhatsNew69Page(FirefoxBasePage):
+
+    URL_TEMPLATE = '/{locale}/firefox/69.0/whatsnew/all/{params}'
+
+    _signed_out_monitor_button_locator = (By.CSS_SELECTOR, '.show-fxa-supported-signed-out .js-monitor-button')
```
Comment (craigcook) [inline tests/pages/firefox/whatsnew/whatsnew_69.py]: `js-fxa-product-button`

File: tests/pages/firefox/whatsnew/whatsnew_69.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    _signed_in_monitor_button_locator = (By.CSS_SELECTOR, '.show-fxa-supported-signed-in .js-monitor-button')
```
Comment (craigcook) [inline tests/pages/firefox/whatsnew/whatsnew_69.py]: `js-fxa-product-button`

File: tests/pages/firefox/whatsnew/whatsnew_70.py [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+from selenium.webdriver.common.by import By
+
+from pages.firefox.base import FirefoxBasePage
+
+
+class FirefoxWhatsNew70Page(FirefoxBasePage):
+
+    URL_TEMPLATE = '/{locale}/firefox/70.0/whatsnew/all/{params}'
+
+    _signed_out_monitor_button_locator = (By.CSS_SELECTOR, '.show-fxa-supported-signed-out .js-monitor-button')
```
Comment (craigcook) [inline tests/pages/firefox/whatsnew/whatsnew_70.py]: `js-fxa-product-button`

File: tests/pages/firefox/whatsnew/whatsnew_70.py [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+    _signed_in_monitor_button_locator = (By.CSS_SELECTOR, '.show-fxa-supported-signed-in .js-monitor-button')
```
Comment (craigcook) [inline tests/pages/firefox/whatsnew/whatsnew_70.py]: `js-fxa-product-button`

**Comment by alexgibson** [comment]
Thanks @craigcook updated (I also pushed to run-integration-tests again for good measure and it passed)

**Pull request merged**